Dockerfile

```diff
@@ -37,13 +37,13 @@ RUN apt-get purge -y -- \
     apt-get -y autoremove
 
 # healthcheck relies on curl, curl relies on part of ssh
-RUN apt install curl -y
```

Comment:
`update` seems redundant with line 5 since no new apt sources are being installed

Reply:
I thought the same thing but because of layering cache the earlier version of curl has been replaced and causes a 404 without this update.

```
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/c/curl/libcurl4_7.68.0-1ubuntu2.18_amd64.deb  404  Not Found [IP: 185.125.190.36 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/c/curl/curl_7.68.0-1ubuntu2.18_amd64.deb  404  Not Found [IP: 185.125.190.36 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Fetched 2,965 kB in 5s (622 kB/s)
```